functions-homework/src/NumberOfDays.java

```diff
@@ -0,0 +1,88 @@
+import javax.swing.JOptionPane;
+import java.util.Calendar;
+import java.util.Scanner;
+
+public class NumberOfDays {
+    private static int calMonth;
+
+    public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
+        int year = 0;
+        int month = 0;
+
+        try {
+            year = Integer.parseInt(JOptionPane.
+                    showInputDialog("Please input year: "));
+        } catch (NumberFormatException e) {
+            JOptionPane.showMessageDialog(null, "Value must be an integer!");
+        }
+        try {
+            month = Integer.parseInt(JOptionPane.
+                    showInputDialog("Please input month (number from 1 to 12): "));
+        } catch (NumberFormatException e) {
+            JOptionPane.showMessageDialog(null, "Value must be an integer!");
+        }
+
+        int days = 0;
+        switch (month) {
```

Comment:
toate la gramada asa ?

separation of concerns, scoate clasa separata cu month days calculator, in care sa ai logica asta (switch) si unde sa ii poti trimite unei metode parametrul cu indexu lunii (javadoc, identare)